src/containers/Nodes/getNodesColumns.tsx

```diff
@@ -184,23 +187,61 @@ const topNodesLoadAverageColumn: NodesColumn = {
 
 const topNodesMemoryColumn: NodesColumn = {
     name: NODES_COLUMNS_IDS.TopNodesMemory,
-    header: 'Memory',
-    render: ({row}) =>
-        row.MemoryUsed ? (
-            <ProgressViewer
-                value={row.MemoryUsed}
-                capacity={row.MemoryLimit}
-                formatValues={formatStorageValuesToGb}
-                colorizeProgress={true}
-            />
-        ) : (
-            '—'
```

Comment:
No data progress bar is more explicit than dash and suits more to neighbour cells and columns.
![Screen Shot 2023-11-21 at 16 59 07](https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/ce5267b7-2c63-4412-a030-4b9a43678583)